float64 xMin
float64 yMin
float64 zMin
float64 xMax
float64 yMax
float64 zMax
